code/game/machinery/lightswitch.dm

```diff
@@ -34,8 +34,12 @@
 /obj/machinery/light_switch/examine(mob/user)
 	. = ..()
 	. += "It is [area.lightswitch ? "on" : "off"]."
+	if(obj_flags & EMAGGED & user.can_hear())
+		. += "You hear a faint hum coming from the switch."
```

Comment:
Message could use some clarification or a danger span
```suggestion
		. += "<span class="danger">You hear a faint hum coming from the switch.</span>"
```

Reply:
No. 
An item humming isn't considered something an IC person would consider dangerous. 
Do you consider a switch dangerous if it gives off a soft hum?
The spans are there to convey IC perceived danger.

Reply:
Might as well make it have no way to know its emagged then, locking things like this to only players who read PRs is a bad thing as it makes new player unable to tell what is bad while more code savvy players can bypass traps

Reply:
While I agree with my original point I am not closed-minded and I can see how this being a regular piece of text can make it unfair to those people per your point. 
There is no middle compromise here (except something like a notice span) so I'll go ahead and fully implement your suggestion.